docs/cheatsheet.rst

```diff
@@ -82,6 +82,8 @@ Global Variables
   the given arguments starting from the second and prepends the given four-byte selector
 - ``abi.encodeWithSignature(string memory signature, ...) returns (bytes memory)``: Equivalent
   to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature)), ...)```
+- ``bytes.concat(...) returns (bytes memory)``: :ref:`Concatenates variable number of
```

Comment:
Can you also add a similar line to https://docs.soliditylang.org/en/v0.8.3/units-and-global-variables.html# ?

Reply:
Oh sorry, it is there.